pegasus/sites.v3/code.org/views/header.haml

```diff
@@ -33,6 +33,8 @@
 
```

Comment:
If I don't have the special cookie, will I still see a hamburger, but it will be the old version?